pkg/data/examples.go

```diff
@@ -1627,6 +1627,71 @@ var ExampleHogwartsSimpler = registerExample(&Example{
 	DisjointGroupsTags: disjointHousesAndFunctionality,
 })
 
+var ExampleHogwartsScopeAnd = registerExample(&Example{
```

Comment:
only one example json file should be added based on this diff.. why are 4 json files added in the `json` dir of this PR?

Reply:
please recheck, it seems like examples that were not added eventually.

Reply:
`ExampleAppWithGroupsAndSegments` is used, but not `ExampleAppWithGroupsAndSegmentsInfer` that was added in this PR..